mathics/builtin/drawing/graphics3d.py

```diff
@@ -462,7 +462,7 @@ def boxes_to_js(self, leaves=None, **options):
         We include enclosing script tagging.
         """
         json_repr = self.boxes_to_json(leaves, **options)
-        js = f'<graphics3d data="{json_repr}"/>'
```

Comment:
Better is to use some sort of triple quoting which almost certainly can't appear inside json_repr

Reply:
@rocky HTML just accept single or double quote. The `json_repr` don't contain double quotes.

Reply:
Not a big deal. I'll adjust it down the line.